common.go

```diff
@@ -120,12 +120,23 @@ func IsReadyPod(pod *apiv1.Pod) bool {
 		return false
 	}
 
+	// pod is not in Running Phase. It can be in Pending,
+	// Succeeded, Failed, Unknown
+	if pod.Status.Phase != apiv1.PodRunning {
```

Comment:
Would you mind adding some comment to describe the problem we saw in pod information?

Reply:
Something like this?
```
	// pod is not in Running Phase. It can be in Pending,
	// Succeeded, Failed, Unknown. In some cases 
       // the pod can be in different sate than Running even 
       // if all the containers are in Ready state
```

Reply:
> Though it's not in the PR change, how about add more information (e.g. pod name, namespace) in error return here

That would be a nice addition, doing that change.

Reply:
How about this? Feel free to change it, if it's not good enough. 
```
// pod is not in Running Phase. It can be in Pending,
// Succeeded, Failed, Unknown. In some cases 
// the pod can be in different sate than Running, for 
// example kubernetes sets a pod to Termination while 
// k8s waits for the grace period of the pod, even if all 
// the containers are in Ready state
```